### DISCUSSION TITLE
Webcam settings

### DISCUSSION THREAD

**Discussion by Avpman2** [xblax/flashforge_ad5m_klipper_mod#172] [Q&A] [answered]
I had the webcam working, but I needed to factory reset my printer, and I reloaded the klipper mod. But now I can't get the cam working despite setting it up in Mainsail. Here is my config. Anything wrong? 

![Screenshot 2024-05-04 184005](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/3275462/5dea1800-2203-4a3e-906a-c8816cde5d67)

**Comment by Avpman2** [comment]
Anyone?

**Comment by Tiwatz** [comment, marked as answer]
This is my config

![Screenshot_20240505_203321_Chrome](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/162510297/68fd4463-425e-4d1b-bfc1-e5eed1997e0d)

**Reply by Avpman2** [reply]
Did it!

**Comment by Avpman2** [comment]
Dude, you ARE the man! 
Thanks!!!!
